voluptuous/tests/tests.py

```diff
@@ -1300,39 +1319,36 @@ def __hash__(self):
     hashable_obj_provided_in_input = HashableObjectWhichWillBeKeyInDict()
 
     with pytest.raises(MultipleInvalid) as ctx:
-        s({hashable_obj_provided_in_input: [0, 1, 'should be int']})
+        s({hashable_obj_provided_in_input: [0, 1, "should be int"]})
     assert ctx.value.errors[0].path == [hashable_obj_provided_in_input, 2]
 
 
 def test_self_any():
-    schema = Schema({"number": int,
-                     "follow": Any(Self, "stop")})
+    schema = Schema({"number": int, "follow": Any(Self, "stop")})
     with pytest.raises(MultipleInvalid) as ctx:
         schema({"number": "abc"})
     assert len(ctx.value.errors) == 1
     assert isinstance(ctx.value.errors[0], TypeInvalid)
 
     with raises(MultipleInvalid):
-        schema({"follow": {"number": '123456.712'}})
+        schema({"follow": {"number": "123456.712"}})
 
     schema({"follow": {"number": 123456}})
     schema({"follow": {"follow": {"number": 123456}}})
     schema({"follow": {"follow": {"number": 123456, "follow": "stop"}}})
 
 
 def test_self_all():
-    schema = Schema({"number": int,
-                     "follow": All(Self,
-                                   Schema({"extra_number": int},
-                                          extra=ALLOW_EXTRA))},
-                    extra=ALLOW_EXTRA)
+    schema = Schema(
+        {"number": int, "follow": All(Self, Schema({"extra_number": int}, extra=ALLOW_EXTRA))}, extra=ALLOW_EXTRA
```

Comment:
this also didn't get any clearer, please expand it again (might be able to get black to do that with a trailing comma)

Reply:
limiting the line length to default 88 did the trick